src/group_theory/group_action.lean

```diff
@@ -169,6 +236,59 @@ mul_action.comp_hom (quotient H) (subtype.val : I → α)
 
 end mul_action
 
+namespace mul_action
+
+section units
+variables [monoid α] [mul_action α β]
+
+instance units_mul_action: mul_action (units α) β :=
+{ smul := λ a b, a.val • b,
+  one_smul := λ b, one_smul α b,
+  mul_smul := λ a₁ a₂ b, mul_smul a₁.val a₂.val b }
+
+variables {α β}
+
+lemma conj_smul (a₁ : units α) (a₂ : α) (b : β) : (a₁.conj a₂) • b = (to_perm (units α) β a₁).conj((•) a₂) b :=
+show (a₁.val * a₂ * a₁.inv) • b = a₁.val • a₂ • a₁.inv • b,
+by simp only [mul_smul]
+
+variables (β)
+
+lemma conj_smul_funeq (a₁ : units α) (a₂ : α) :
+  ((•) (a₁.conj a₂) : β → β) = (to_perm (units α) β a₁).conj((•) a₂) :=
+funext $ conj_smul a₁ a₂
+
+variables {β}
+
+lemma mem_stabilizer_smul_iff (u : units α) (x : α) (b : β) :
```

Comment:
I suggest changing the name of this to `conj_mem_stablizer_smul_iff`, and reversing the order of the `iff`. Usually the more complicated expression goes on the left hand side.